lms/djangoapps/teams/tests/test_views.py

```diff
@@ -386,9 +431,21 @@ def test_page(self):
         self.assertIsNone(result['next'])
         self.assertIsNotNone(result['previous'])
 
-    def test_expand_user(self):
+    def test_expand_private_user(self):
         result = self.get_teams_list(200, {'expand': 'user', 'topic_id': 'topic_0'})
```

Comment:
We don't have to specify a particular user because all users are "private" (no year_of_birth set)?